django/contrib/admin/static/admin/js/admin/DateTimeShortcuts.js

```diff
@@ -389,11 +389,13 @@
         handleCalendarCallback: function(num) {
             let format = get_format('DATE_INPUT_FORMATS')[0];
             // the format needs to be escaped a little
-            format = format.replace('\\', '\\\\')
-                .replace('\r', '\\r')
-                .replace('\n', '\\n')
-                .replace('\t', '\\t')
-                .replace("'", "\\'");
+            // this \\ doesn't work with replace or replaceall, results in error thrown by init test
+            format = format.replace(/\\/g, "\\\\");
+
+            format = format.replace(new RegExp('\r', 'g'), '\\r')
+                .replace(new RegExp('\n', 'g'), '\\n')
+                .replace(new RegExp('\t', 'g'), '\\t')
+                .replace(new RegExp("'", 'g'), "\\'");
```

Comment:
We can keep one expression as it was before, no need to reassign `format`
```suggestion
            format = format.replace(/\\/g, "\\\\")
                .replace(new RegExp('\r', 'g'), '\\r')
                .replace(new RegExp('\n', 'g'), '\\n')
                .replace(new RegExp('\t', 'g'), '\\t')
                .replace(new RegExp("'", 'g'), "\\'");
```